.travis.yml

```diff
@@ -15,8 +15,6 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 
-sudo: required
```

Comment:
When we running docker, we need the sudo right.
Why do you remove that part.

Reply:
https://changelog.travis-ci.com/the-container-based-build-environment-is-fully-deprecated-84517